index.html

```diff
@@ -15,35 +15,33 @@
   </section>
 
   <section class = "players-container container">
+
     <article class="player player-1"> 
```

Comment:
Not totally sure if `articles` are the most semantic HTML tag here. Usually articles are reserved for news/blog sites and denote that the content inside of it is some sort of journalistic feat :)